src/progly/CMakeLists.txt

```diff
@@ -1,3 +1,10 @@
+# --------------------------------- #
```

Comment:
can you just add a comment here please that this is not required for skyhook so can be removed later after paper completed.

Reply:
added comment.